restcomm/restcomm.telephony/src/main/java/org/restcomm/connect/telephony/CallManager.java

```diff
@@ -702,72 +703,6 @@ public void run() {
 
     }
 
-    /**
-     * FIXME: duplicated code make into static function or something more optimized
-     * Replace headers
-     *
-     * @param SipServletRequest message
-     * @param Map<String,       ArrayList<String> > headers
-     */
-    private void addHeadersToMessage(SipServletRequest message, Map<String, ArrayList<String>> headers) {
```

Comment:
i take this was duplicated from Call logic, exactly same code?

Reply:
The only difference being the two were referring to private `sipFactory`, which we added as a param in the relocated one